shared/src/main/java/com/couchbase/lite/AbstractReplicator.java

```diff
@@ -371,6 +433,44 @@ public void removeChangeListener(ListenerToken token) {
         }
     }
 
+    /**
+     * Set the given DocumentReplicatedListener to the this replicator.
+     *
+     * @param listener
+     * @return ListenerToken A token to remove the handler later
+     */
+    public ListenerToken addReplicationListener(DocumentReplicatedListener listener) {
```

Comment:
The correct signature is `addDocumentReplicationListener(DocumentReplicationListener listener)`.

Reply:
DocumentReplicatedListener is listen for document replicated event, so shouldn't DocumentReplicatedListener be better?